sql/core/src/test/resources/sql-tests/inputs/show-tables-legacy.sql

```diff
@@ -0,0 +1,2 @@
+--SET spark.sql.legacy.keepCommandOutputSchema=true
```

Comment:
This might be useful to verify the legacy schema

Reply:
Is it relevant to this PR's `isView` column addition?

Reply:
the  code change is related, `spark.sql.legacy.keepCommandOutputSchema=true` will omit the `isView` column

Reply:
For instance, 
https://github.com/apache/spark/pull/31804/files#diff-a854c4d2d3a463feef2307548a917bae452850a16de730f8a14f84a4eb79a16fR64

https://github.com/apache/spark/pull/31804/files#diff-b6f30759017988fd0963ce840918f541caf610a1793fa73f17e61b03a2acb797R64